client-ui/src/app/components/time-frame-picker/date-picker/date-picker.component.html

```diff
@@ -14,24 +14,7 @@
   ~ limitations under the License.
   -->
 
-<form class="form-inline">
-  <div class="form-group">
-    <div class="input-group">
-      <input name="dp" ngbDatepicker
-             class="form-control" [class.is-invalid]="!valid"
```

Comment:
I see `[class.is-invalid]="!valid"` is missing in your variant. Is it intentional? It was there for visual indication of invalid input.